frontend/src/types/sql-review-schema.yaml

```diff
@@ -21,6 +21,7 @@ ruleList:
       - TIDB
       - POSTGRES
       - ORACLE
+      - DM
```

Comment:
The change should be associated with SQL review change in Go files. Maybe we can hold the change in this file for now.